mopidy/internal/deps.py

```diff
@@ -109,15 +112,14 @@ def pkg_info(
                     not include_extras and "extra" in raw
                 ):
                     continue
-                entry = re.match(
-                    "[a-zA-Z0-9_']+", raw
```

Comment:
@janiversen I assume we don't actually have dependencies with `'` in their name?

Reply:
Correct.

But I am a bit surprised that the lib do not work for 3.10, I used 3.10 without problems earlier this week. Of course I have a simple installation and that might be the reason I did not see the loop.

Reply:
The lib works for 3.10, I just want to keep the dependencies as lean as possible. This way, we also remember to remove the dependency when we require 3.10+ someday.